detectron2/config/defaults.py

```diff
@@ -158,7 +158,7 @@
 _C.MODEL.PROPOSAL_GENERATOR.NAME = "RPN"
 # Proposal height and width both need to be greater than MIN_SIZE
 # (a the scale used during training or inference)
-_C.MODEL.PROPOSAL_GENERATOR.MIN_SIZE = 0
+_C.MODEL.PROPOSAL_GENERATOR.MIN_SIZE = 0.0
```

Comment:
I think this may break an old config, when trying to load an old config file that has "0". `yacs` checks types of values and may complain about integer vs. float.

Reply:
done